t/core/config_etcd.t

```diff
@@ -233,44 +233,7 @@ passed
 
 
 
-=== TEST 7: ensure only one auth request per subsystem for all the etcd sync
---- yaml_config
-apisix:
-  node_listen: 1984
-deployment:
-  role: traditional
-  role_traditional:
-    config_provider: etcd
-  etcd:
-    host:
-      - "http://127.0.0.1:1980" -- fake server port
-    timeout: 1
-    user: root                    # root username for etcd
-    password: 5tHkHhYkjr6cQY      # root password for etcd
---- extra_init_by_lua
-local health_check = require("resty.etcd.health_check")
-health_check.get_target_status = function()
-    return true
-end
---- config
-    location /t {
-        content_by_lua_block {
-            ngx.sleep(0.5)
-        }
-    }
---- request
-GET /t
---- grep_error_log eval
-qr/etcd auth failed/
---- grep_error_log_out
-etcd auth failed
```

Comment:
no duplicated auth anymore, due to cli reused.

Reply:
Why remove this? how do you prove that the cli is reused?

Reply:
No need to prove reused or not in this test case.
If we need a meaningless test case, we could hook the get_etcd_cli to check if the cli is reused, but we could prove it in reality based on the fact that the performance is improved, which is hard to express via test::nginx. So we just do not write a new test case for this PR.

Back to this test case, it is arbitrary and does not describe what is "subsystem".
It's useless and confusing. If you could explain it, then I will keep it.

Reply:
We need a test case to make sure the fix work as expected. :)
Plz add a new test case to prove it.

Reply:
BTW, it‘s weird to delete a test case in a fix PR without adding any new test case.

Reply:
done